src/payment/braintree.md

```diff
@@ -82,6 +82,8 @@ Go to [Braintree Payments][1] and sign up for an account.
 
    Ensure the Advanced Fraud Protection is enabled in the Settings/Protection section of your account.
 
+1. Set the `Bypass Fraud Protection Threshold` so that the `Advanced Fraud Protection` checks are bypassed, if the threshold is met or exceeded. Leaving this field blank will disable this option.
```

Comment:
```suggestion
1. Set the `Bypass Fraud Protection Threshold` so that the `Advanced Fraud Protection` checks are bypassed when the threshold is met or exceeded.

   Leaving this field blank will disable this option.
```